README.md

```diff
@@ -56,6 +56,7 @@ Options:
                                   deletes any files found in there. (If you
                                   restore the photo in iCloud, it will be
                                   downloaded again.)
+  --auto-delete-downloaded        Auto delete downloaded photo from icloud.
```

Comment:
What can we do to help users avoid running `--auto-delete-downloaded` and `--auto-delete` together or consequently (which will delete all photos locally and in the cloud)?